nkowaokwu/igbo_api
# Fix signup form submission failure

## Describe your changes
Pass down the `field` props from the `useForm` hook's controller into the `input box`. This helps track changes in the `input box` value and updates the errors object if the `input box` value doesn't satisfy specified rules.


## Issue ticket number and link
Closed #776 

## Motivation and Context
The PR fixes the difficulty experienced when trying to sign up as a developer

## How Has This Been Tested?
1. On your local environment navigate to `/signup` page
2. Fill up the details required and click the `Create account` button
3. Notice signup is successful and  API key generated

## Screenshots (if appropriate):
Before
![image](https://github.com/nkowaokwu/igbo_api/assets/32741303/f5ad3d58-02d8-4847-9c7d-2f404b5c807c)

After
![Screenshot 2024-03-11 at 09 37 47](https://github.com/nkowaokwu/igbo_api/assets/32741303/a4faa36b-59c3-403b-bf07-66236e7eed9e)
